.github/workflows/test-issue.yml

```diff
@@ -0,0 +1,17 @@
+name: TEST_ISSUE
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  run-this-action:
+    name: Run action
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
would you mind giving it a try without checking it out, or is that strictly needed? why?
```suggestion
```

Reply:
to run the action from a branch with the locally made changes

Reply:
ah, understood